app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [Optional] You may also be considered adding foreign_keys, that's

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
Hello @FrederickMih, thank you for the review. It's already added in `foreign_key: 'author_id'` no? 😅

Reply:
Hello @FrederickMih, thank you for the review! I think the user foreign keys should be added in the `posts`, `comments`, and `likes` models not in `users`. Please correct me if I'm wrong.